### PR TITLE
Custom contact reasons

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

**Comment by helenb** [comment]
@chris-lawton, Nathan added these updates over the weekend. They appear to relate to this commit in the wagtail build: https://github.com/torchbox/wagtail-torchbox/commit/0ca994e49e2ec5aa4bb6033f6bfdd6dde9b00a50

File: src/components/contact-detailed/contact.js [hunk 1]
```diff
@@ -18,31 +18,32 @@ const Contact = ({
...
-                    {reason.title}
-                  </h2>
-                  <p className={styles.contactBlockReasonsItemText}>
-                    {reason.description}
-                  </p>
-                </div>
-              </li>
-            ))}
-          </ul>
+          <h1>{heading}</h1>
```
Comment (chris-lawton) [inline src/components/contact-detailed/contact.js]: A global search of the repo shows that this is the only `<h1>` - I think axe will moan on other pages saying `Page must contain a level-one heading` so may be worth addressing in another PR?

Comment (helenb) [reply src/components/contact-detailed/contact.js]: I think all the streamfield blocks use h1 (and are valid because within a section). I am just pondering the heading structure and might want a second opinion! Certainly needs improving.

**Comment by helenb** [comment]
@NathHorrigan @chris-lawton can this be merged to develop now?

**Comment by chris-lawton** [comment]
Looks good to me.

**Comment by helenb** [comment]
@NathHorrigan please resolve conflicts and merge when you are ready.
